system/handlers/admin/AuditTrail.cfc

```diff
@@ -0,0 +1,44 @@
+component extends="preside.system.base.AdminHandler" output=false {
+
+	public void function preHandler( event ) output=false {
+		super.preHandler( argumentCollection=arguments );
+
+		_permissionsCheck( "navigate", event );
+
+		prc.pageIcon = "fa-history";
+		event.addAdminBreadCrumb(
+			  title = translateResource( "cms:auditTrail.breadcrumbTitle" )
+			, link  = event.buildAdminLink( linkTo="auditTrail" )
+		);
+	}
+
+	public void function index( event, rc, prc ) output=false {
+		prc.logs         = getModel( "AuditService" ).getAuditLog(1,10);
+		prc.pageTitle    = translateResource( "cms:auditTrail.page.title" );
+		prc.pageSubTitle = translateResource( "cms:auditTrail.page.subtitle" );
+	}
+
+	public any function loadMore(event, rc, prc) output=false {
+		prc.logs         = getModel( "AuditService" ).getAuditLog(rc.start,10);
```

Comment:
We need to be more _defensive_ with the `rc.start` here. Whenever there is raw user input, we need to ensure it is as expected. Also, please use "page", rather than start. So:

```
var page = Val( rc.page ?: 2 );
prc.logs = getModel( "AuditService" ).getAuditLog( page, 10 );
```